app/api.py

```diff
@@ -468,4 +484,20 @@ def status():
     return {"status": "ok"}
 
 
+# Route only available in dev mode
+# This route is used to set the session cookie for the auth server
+class SessionBody(BaseModel):
+    session: str
+
+
+auth_server_static = os.getenv("AUTH_SERVER_STATIC")
+if auth_server_static and auth_server_static != "":
+
+    @api_v1_router.post("/set_session_cookie")
+    def set_session_cookie(request: Request, body: SessionBody):
+        response = PlainTextResponse("Session cookie set")
+        response.set_cookie(key="session", value=body.session)
+        return response
```

Comment:
Can you add a docstring with documentation on how to use it… this way it will display in /docs where people use it.

Also add an explanation about it in README.md